include/eosio/chain_conversions.hpp

```diff
@@ -298,7 +298,7 @@ inline std::string microseconds_to_str(uint64_t microseconds) {
       return false;
    if (!parse_uint(sec, 2))
       return false;
-   result = sys_days(year_month_day{year_t{y}, month_t{m}, day_t{d}}.to_days()).time_since_epoch().count() * 86400 + h * 3600 + min * 60 + sec;
+   result = sys_days(year_month_day{year_t{y}, month_t{m}, day_t{d}}.to_days()).time_since_epoch().count() * 86400u + h * 3600u + min * 60u + sec;
```

Comment:
`sys_days(year_month_day{year_t{y}, month_t{m}, day_t{d}}.to_days()).time_since_epoch().count()` actually returns an `int`, so when testing at extremities this resulted in `49710 * 86400` overflowing an integer.

Reply:
Should we add an overflow check?

Reply:
It does seem like there should be some input validation in here in case someone tries to send through a string from JSON with year 3123, or even a month greater than 12 I think it accepted.

But, this now does handle all "supported" cases up to and including the 0xffffffff value; it was this test that hit the issue,
https://github.com/AntelopeIO/abieos/blob/815fca5fb3cd632eaffff098b14809a99229bebc/src/template_test.cpp#L198

So, I think any potential input validation would be best left for a separate improvement.